src/Components/ResultPage/ResultPage.test.js

```diff
@@ -0,0 +1,224 @@
+        expect(cardDetails).toBeInTheDocument()
+        
+        
+      });
+
+      it('It should display information about other activities', () => {
+        render(
+            <MemoryRouter>
+                <ResultPage
+                    name={name}
+                    data={data}
+                    randomActivity={randomActivity}
+                    determineRandomActivity={determineRandomActivity}
+                    error={''}
+                />
+            </MemoryRouter>
+        )
+
+        const cardTitle = screen.getByText('The Witches');
+        const cardDate = screen.getByRole('heading', { name: /2020\-10\-22/i })
+        const cardDetails = screen.getByText( 'AdventureComedyFamilyFantasyHorror' )
+
+        expect(cardTitle).toBeInTheDocument()
+        expect(cardDate).toBeInTheDocument()
+        expect(cardDetails).toBeInTheDocument()
+        
+      });
+      
+      it('When the user click on the skip button, a new activity should display on the page', async() => {
+        render(
+            <MemoryRouter>
+                <ResultPage
+                    name={name}
+                    data={data}
+                    randomActivity={randomActivity}
+                    determineRandomActivity={determineRandomActivity}
+                    error={''}
+                />
+            </MemoryRouter>
+        )
+        
+        const banner = screen.getByRole('heading', { name: /finally, here is a suggestion for you!/i })
+        const movieTitle = screen.getByText('Harry Potter and the Prisoner of Azkaban');
+        const moviePublic = screen.getByText('PG');
+        const randomImg = screen.getByRole('img');
+        const simmilarPicksTitle = screen.getByRole('heading', { name: /simmilar picks/i });
+        const choosemeButton = screen.getByRole('button', { name: /choose me!/i });
+        const skipButton = screen.getByRole('button', { name: /skip/i });
+
+        expect(banner).toBeInTheDocument();
+        expect(movieTitle).toBeInTheDocument();
+        expect(moviePublic).toBeInTheDocument();
+        expect(randomImg).toBeInTheDocument();
+        expect(simmilarPicksTitle).toBeInTheDocument();
+        expect(choosemeButton).toBeInTheDocument();
+        expect(skipButton).toBeInTheDocument();
+        expect(choosemeButton).toBeInTheDocument();
+
+        userEvent.click(skipButton);
+        expect(determineRandomActivity).toHaveBeenCalled();
+
+        const movieTitle2  =  await waitFor(() => screen.getByText('Corpse Bride'))
+        const moviePublic2  =  await waitFor(() => screen.getByText('PG'))
+        const randomImg2 = await waitFor(() => screen.getByRole('img'));
+        const simmilarPicksTitle2 = screen.getByRole('heading', { name: /simmilar picks/i });
+        const choosemeButton2 = screen.getByRole('button', { name: /choose me!/i });
+        const skipButton2 = screen.getByRole('button', { name: /skip/i });
+
+        expect(movieTitle2).toBeInTheDocument();
```

Comment:
Amazing testing here, very thorough and thanks for covering the edge cases we've implemented.